### ISSUE TITLE
Training output

### ISSUE THREAD

**Issue by ghost** [Hephyrius/Binance-Ai-Bot-Starter#1]
Hi, 
I think you are actively working on developing a machine learning bot on binance as I do now. I would like to collaborate if I can because I was looking for an example to implement some openai-gym models  and your bot gives me a start point. As far as I see there is no backtesting in your bot, before going live how do you evaluate the model? Training script gives out the output below, how do you decide on what to trade with this data?
Accuracy = 0.7169811320754716
MSE = 0.2830188679245283
False Pos = 0.12578616352201258
False Neg = 0.15723270440251572

**Comment by Hephyrius** [comment]
The metrics in the script are supposed to give you a basic overview at a glance. So you can decide if you wanted to test the model or work on further model tweaks or  feature creation/selection. 

I have a very simple backtest script I use. Essentially I use the binance library to get minutely data for the period my models haven't been trained on, the validation set. I then cycle through this data and do paper buys and sells every hour or so with the hourly dats, with a trailing stoploss taking into account the minutely highs and lows. From this I calculate stats like drawdown, alpha,  number of trades and other useful metrics. 

There are also tons of python libraries that allow you to do back testing if you provide them with formatted data. So that's always an option. 

I would definitely like to collaborate on an open ai style environment for this kind of trading. It would be nice to see how well reinforcement learning works on trading. I've read papers that suggest it's pretty bad, but it would be nice to judge that for myself :)
